tests/Temporalio.Tests/WorkflowEnvironment.cs

```diff
@@ -34,6 +34,7 @@ public async Task InitializeAsync()
                 },
             },
         });
+        // env = new(await Temporalio.Client.TemporalClient.ConnectAsync(new("localhost:7233")));
```

Comment:
leftover?

Reply:
Intentionally so, because I come here and run against local server so I have a UI (I used it to download those JSONs for the replayer tests). But I can make a comment to that effect.